### ISSUE TITLE
"You can't set the tabs viewControllers multiple times."

### ISSUE THREAD

**Issue by Gargo** [Abedalkareem/AMTabView#6]
Just WTF? Why is it not described in readme? What if I need to show "set1 of view controllers" to anonymous user and "set2" if the user is logged in?

**Comment by Abedalkareem** [comment]
Yes right now it does not support that, maybe in the future I'll add it.
